BedrockServer.cpp

```diff
@@ -1701,27 +1703,27 @@ void BedrockServer::_status(BedrockCommand& command) {
     SData& request  = command.request;
     SData& response = command.response;
 
-    // We'll return whether or not this server is slaving.
-    if (SIEquals(request.methodLine, STATUS_IS_SLAVE)) {
+    // We'll return whether or not this server is following.
+    if (SIEquals(request.methodLine, STATUS_IS_SLAVE) || SIEquals(request.methodLine, STATUS_IS_FOLLOWER)) {
         // Used for liveness check for HAProxy. It's limited to HTTP style requests for it's liveness checks, so let's
         // pretend to be an HTTP server for this purpose. This allows us to load balance incoming requests.
         //
         // HAProxy interprets 2xx/3xx level responses as alive, 4xx/5xx level responses as dead.
         SQLiteNode::State state = _replicationState.load();
-        if (state == SQLiteNode::SLAVING) {
-            response.methodLine = "HTTP/1.1 200 Slaving";
+        string verbing = SIEquals(request.methodLine, STATUS_IS_FOLLOWER) ? "Following" : "Slaving";
+        if (state == SQLiteNode::FOLLOWING) {
+            response.methodLine = "HTTP/1.1 200 "+ verbing;
         } else {
-            response.methodLine = "HTTP/1.1 500 Not slaving. State="
-                                  + SQLiteNode::stateNames[state];
+            response.methodLine = "HTTP/1.1 500 Not " + verbing + ". State=" + SQLiteNode::stateName(state);
         }
     }
 
     else if (SIEquals(request.methodLine, STATUS_HANDLING_COMMANDS)) {
         // This is similar to the above check, and is used for letting HAProxy load-balance commands.
         SQLiteNode::State state = _replicationState.load();
-        if (state != SQLiteNode::SLAVING) {
-            response.methodLine = "HTTP/1.1 500 Not slaving. State=" + SQLiteNode::stateNames[state];
-        } else if (_version != _masterVersion.load()) {
+        if (state != SQLiteNode::FOLLOWING) {
+            response.methodLine = "HTTP/1.1 500 Not following. State=" + SQLiteNode::stateName(state);
+        } else if (_version != _leaderVersion.load()) {
             response.methodLine = "HTTP/1.1 500 Mismatched version. Version=" + _version;
         } else {
             response.methodLine = "HTTP/1.1 200 Slaving";
```

Comment:
Yes, because I don't know what is relying on this. When we're sure nobody is looking for "200 Slaving" and is just looking for "200", we can remove it.